test/solveEquation/solveEquation.test.js

```diff
@@ -14,8 +14,9 @@ function testSolve(equationString, outputStr, debug=false) {
   else {
     lastStep = steps[steps.length -1].newEquation.print();
   }
-  it(equationString + ' -> ' + outputStr, () => {
+  it(equationString + ' -> ' + outputStr, (done) => {
     assert.equal(lastStep, outputStr);
+    done();
```

Comment:
Ya right ! But normally if am running the test on my system I get an error indicating `timeout of  2000ms exceeded`. However errors are gone if I treat it as async or in shot if I use done() callback.